crates/interpreter/src/exec.rs

```diff
@@ -192,10 +194,10 @@ impl<'m> Store<'m> {
             let x = parser.parse_funcidx().into_ok();
             let ptr = self.func_ptr(inst_id, x);
             let inst_id = ptr.instance().unwrap_wasm();
-            let mut parser = self.insts[inst_id].module.func(ptr.index());
+            let (mut parser, side_table) = self.insts[inst_id].module.func(ptr.index());
             let mut locals = Vec::new();
             append_locals(&mut parser, &mut locals);
-            let thread = Thread::new(parser, Frame::new(inst_id, 0, &[], locals));
+            let thread = Thread::new(parser, Frame::new(inst_id, 0, &[], locals, side_table));
             let result = thread.run(self)?;
```

Comment:
```
error[E0502]: cannot borrow `*self` as mutable because it is also borrowed as immutable
   --> src/exec.rs:201:26
    |
107 | impl<'m> Store<'m> {
    |      -- lifetime `'m` defined here
...
197 |             let (mut parser, side_table) = self.insts[inst_id].module.func(ptr.index());
    |                                            --------------------------------------------
    |                                            |
    |                                            immutable borrow occurs here
    |                                            argument requires that `self.insts` is borrowed for `'m`
...
201 |             let result = thread.run(self)?;
    |                          ^^^^^^^^^^^^^^^^ mutable borrow occurs here

```